code/datums/wires/simple_wire.dm

```diff
@@ -0,0 +1,22 @@
+/datum/wires/simple_wire   //two bit wire type just to add assemblies to things that dont have specific wires
```

Comment:
Assembly code has too much checks for wires, i thought its easier to just make assemblyholder a special subtype of wire, they have exactly nothing in cimmin with wires apart from type and being compatible with assemblies